CMakeLists.txt

```diff
@@ -80,8 +80,11 @@ endif()
 #
 option(fletch_BUILD_WITH_PYTHON "Build with Python support where appropriate" FALSE)
 if (fletch_BUILD_WITH_PYTHON)
+
   find_package(PythonInterp 2.7 REQUIRED)
   find_package(PythonLibs 2.7 REQUIRED)
+  find_package(NumPy 1.7)
```

Comment:
If numpy is required for certain packages, the find_package should specify REQUIRED in those cases. I guess there are 2 options I can think of. First is to have an if in here which asks if Caffe is enabled, then add REQUIRED to find_package. The other would be to add the find_package to the External_Caffe.cmake file with the REQUIRED option in there. Not surer which would be cleaner or if there is another option I haven't considered. Either way, we need a hard-stop during configure if we selected something which requires a missing numpy.